internal/cli/task_cancel.go

```diff
@@ -0,0 +1,103 @@
+package cli
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/pkg/server/gen"
+)
+
+type TaskCancelCommand struct {
+	*baseCommand
+
+	flagRunJobId string
+}
+
+func (c *TaskCancelCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+	); err != nil {
+		return 1
+	}
+	ctx := c.Ctx
+
+	var taskId string
+	if len(c.args) == 0 && c.flagRunJobId == "" {
+		c.ui.Output("Task ID or Run Job Id required.\n\n%s", c.Help(), terminal.WithErrorStyle())
+		return 1
+	} else {
+		taskId = c.args[0]
+	}
```

Comment:
```suggestion
	if len(c.args) == 0 && c.flagRunJobId == "" {
		c.ui.Output("Task ID or Run Job Id required.\n\n%s", c.Help(), terminal.WithErrorStyle())
		return 1
	}
	taskId := c.args[0]
```

Nit: remove the unnecessary `else` block